core/src/main/java/org/apache/accumulo/core/cli/ClientOpts.java

```diff
@@ -82,6 +68,78 @@ public ColumnVisibility convert(String value) {
     }
   }
 
+  public static class PasswordConverter implements IStringConverter<String> {
+    public static final String STDIN = "stdin";
+
+    private enum KeyType {
+      PASS("pass:"), ENV("env:") {
+        @Override
+        String process(String value) {
+          return System.getenv(value);
+        }
+      },
+      FILE("file:") {
+        @SuppressFBWarnings(value = "PATH_TRAVERSAL_IN",
+            justification = "app is run in same security context as user providing the filename")
+        @Override
+        String process(String value) {
+          Scanner scanner = null;
+          try {
+            scanner = new Scanner(new File(value), UTF_8);
+            return scanner.nextLine();
+          } catch (IOException e) {
+            throw new ParameterException(e);
+          } finally {
+            if (scanner != null) {
+              scanner.close();
+            }
+          }
+        }
+      },
+      STDIN(PasswordConverter.STDIN) {
+        @Override
+        public boolean matches(String value) {
+          return prefix.equals(value);
+        }
+
+        @Override
+        public String convert(String value) {
+          // Will check for this later
```

Comment:
Would you happen to know where this processed?  I didn't see it.  If you are not sure I can look further.

Reply:
If I understand it correctly stdin is not a prefix, more like a placeholder. If the password is exactly "stdin" Shell.java at line 332 checks it and reads the actual bare password from console.